src/class-drip-woocommerce-cart-events.php

```diff
@@ -171,4 +175,27 @@ private function generate_drip_cart_session_id() {
 		$random_data = random_bytes( 32 ); // as of php7, random_bytes is advertised as cryptographically secure.
 		return hash( 'sha256', $random_data );
 	}
+
+	private function find_drip_visitor_uuid() {
```

Comment:
This method seems like a good opportunity to split out a simple cookie parser class with some unit tests… We have the ability to run PHPUnit tests as part of the build instead of doing Cypress tests. Check out the `tests/` folder…

Reply:
I took a crack at this in https://github.com/DripEmail/drip-woocommerce/pull/18. Feel free to merge into this PR if you like that.